release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -115,7 +115,12 @@ To install a CSI driver on a cluster running on vSphere, you must have the follo
 * vSphere version 6.7 Update 3 or greater
 * VMware ESXi version 6.7 Update 3 or greater
 
-These components are still supported, but are deprecated; support for them will be removed in a future version of {product-title}.
+Components lower than those above are still supported, but are deprecated. Support for them will be removed in a future version of {product-title}.
```

Comment:
s/Components lower/Components with versions lower  or of versions. 

_Components lower_ doesn't sound right.

Reply:
Done!